mala/common/physical_data.py

```diff
@@ -242,7 +242,7 @@ def read_dimensions_and_dtype_from_numpy_file(self, path):
         loaded_array = np.load(path, mmap_mode="r")
         return self._process_loaded_dimensions(np.shape(loaded_array)), loaded_array.dtype
 
-    def read_dimensions_from_openpmd_file(self, path):
+    def read_dimensions_from_openpmd_file(self, path, comm=None):
```

Comment:
Using a parameter here on purpose and not `get_comm()`, as this makes it obvious at the call site that the function is collective.